### ISSUE TITLE
PM-30 content structure doesn't match PDF 

### ISSUE THREAD

**Issue by AdamOlineDOJ** [usnistgov/oscal-content#56]
# Describe the bug

The structure of control PM-30 in the OSCAL content does not match the structure in the published 800-53 Rev5 PDF file.

PDF structure

![image](https://user-images.githubusercontent.com/64212840/110137119-18d34f00-7d96-11eb-8ab0-a4a772aef7cc.png)

JSON "parts" segment

            "parts": [
              {
                "id": "pm-30_smt",
                "name": "statement",
                "parts": [
                  {
                    "id": "pm-30_smt.a",
                    "name": "item",
                    "props": [
                      {
                        "name": "label",
                        "value": "a."
                      }
                    ],
                    "prose": "Develop an organization-wide strategy for managing supply chain risks associated with the development, acquisition, maintenance, and disposal of systems, system components, and system services;",
                    "parts": [
                      {
                        "id": "pm-30_smt.a.1",
                        "name": "item",
                        "props": [
                          {
                            "name": "label",
                            "value": "1."
                          }
                        ],
                        "prose": "Implement the supply chain risk management strategy consistently across the organization; and",
                        "parts": [
                          {
                            "id": "pm-30_smt.a.1.a",
                            "name": "item",
                            "props": [
                              {
                                "name": "label",
                                "value": "(a)"
                              }
                            ],
                            "prose": "Review and update the supply chain risk management strategy on {{ pm-30_prm_1 }} or as required, to address organizational changes."
                          }
                        ]
                      }
                    ]
                  }
                ]
              },

XML content has similar structure as JSON.

# Who is the bug affecting?

Consumers of the OSCAL content

**Comment by iMichaela** [comment]
@wendellpiex - It appears that the same error listed above is exhibited (for now obvious reason) in the OSCAL XML version too. Not sure why the extracting tool is misinterpreting the original SP 800-53 R5 content:

```
<part name="statement" id="pm-30_smt">
        <part name="item" id="pm-30_smt.a">
          <prop name="label">a.</prop>
          <p>Develop an organization-wide strategy for managing supply chain risks associated with the development, acquisition, maintenance, and disposal of systems, system components, and system services;</p>
          <part name="item" id="pm-30_smt.a.1">
            <prop name="label">1.</prop>
            <p>Implement the supply chain risk management strategy consistently across the organization; and</p>
            <part name="item" id="pm-30_smt.a.1.a">
              <prop name="label">(a)</prop>
              <p>Review and update the supply chain risk management strategy on <insert param-id="pm-30_prm_1"/> or as required, to address organizational changes.</p>
            </part>
          </part>
        </part>
      </part>
```

The correct representation should be:

```
      <part name="statement" id="pm-30_smt">
        <part name="item" id="pm-30_smt.a">
          <prop name="label">a.</prop>
          <p>Develop an organization-wide strategy for managing supply chain risks associated with the development, acquisition, maintenance, and disposal of systems, system components, and system services;    </p>
        </part> 
        <part name="item" id="pm-30_smt.a.1">
          <prop name="label">1.</prop>
          <p>Implement the supply chain risk management strategy consistently across the organization; and</p>
        </part> 
        <part name="item" id="pm-30_smt.a.1.a">
          <prop name="label">(a)</prop>
          <p>Review and update the supply chain risk management strategy on <insert param-id="pm-30_prm_1"/> or as required, to address organizational changes.</p>
        </part> 
      </part>
```
 
The error in the XML is then propagated by the converter into JSON. 

The correct representation in JSON should be:

```
        "parts": [
          {
            "id": "pm-30_smt",
            "name": "statement",
            "parts": [
              {
                "id": "pm-30_smt.a",
                "name": "item",
                "props": [
                  {
                    "name": "label",
                    "value": "a."
                  }
                ],
                "prose": "Develop an organization-wide strategy for managing supply chain risks associated with the development, acquisition, maintenance, and disposal of systems, system components, and system services;" 
                  },
                  {
                    "id": "pm-30_smt.b",
                    "name": "item",
                    "props": [
                      {
                        "name": "label",
                        "value": "b."
                      }
                    ],
                    "prose": "Implement the supply chain risk management strategy consistently across the organization; and" 
                     },
                      {
                        "id": "pm-30_smt.c",
                        "name": "item",
                        "props": [
                          {
                            "name": "label",
                            "value": "c"
                          }
                        ],
                        "prose": "Review and update the supply chain risk management strategy on {{ pm-30_prm_1 }} or as required, to address organizational changes."
                      }
                    ]
```

**Comment by wendellpiez** [comment]
This is a data acquisition error made by the Word->OSCAL converter. So it is erroneous in the OSCAL and its derivative data including the FISMA 'starter' load file. So it must be corrected both in the repo and now upstream (in the FISMA database).

@AdamOlineDOJ thanks for reporting this -- it's a good catch. The lapse appears to be due to the Word inputs for this control alone - the list items in the source (for whatever reason) indicate no indent level (unlike others in similar positions), leading to the converter to nest the list items incorrectly. Fortunately it turns out only one of the controls has this problem (one reason it evaded detection).

Next:
- [ ] amend the data with the correction in the repo
- [ ] coordinate w/ FISMA team to receive update

**Comment by wendellpiez** [comment]
PR is up; next we coordinate this.

**Comment by wendellpiez** [comment]
Also added to this PR a Schematron rule to detect solitary list items in structured statements (i.e. nested "item" parts inside statements without any sibling "item" parts). Runs clean.

**Comment by AdamOlineDOJ** [comment]
Great, thanks for the updates and fix @wendellpiez. Credit goes to John Granata on our team at DOJ for identifying this one.

**Comment by wendellpiez** [comment]
You're welcome @AdamOlineDOJ and thanks to John and everyone helping us make this work.

**Comment by wendellpiez** [comment]
This Issue can be closed - correction was merged with #57
